web/modules/custom/solution/src/SolutionAffiliationFieldItemList.php

```diff
@@ -169,4 +206,61 @@ protected function solutionInOfficialGraph(): bool {
     return $this->solutionInOfficialGraph;
   }
 
+  /**
+   * Returns the graph ids that the collection exists in.
+   *
+   * @return array
+   *   An array of graph URIs that the collection exists in indexed by graph id.
+   */
+  protected function getAvailableGraphs(): array {
+    $graph_handler = $this->getGraphHandler();
+    $graph_ids = $graph_handler->getEntityTypeGraphIds('rdf_entity');
```

Comment:
Shouldn't we restrict to the official graphs? This implementation grabs also the `staging` graph but there's already a policy in place that for non-official graphs the code that use them is responsible with ensuring the solution <-> collection relation.